k8s/prod/common/ccd/data-store-api.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       image: hmctspublic.azurecr.io/ccd/data-store-api:prod-617aef96
       environment:
         IDAM_USER_URL: "https://hmcts-access.service.gov.uk"
-        CCD_DM_DOMAIN: ^https?://(?:api-gateway\.dm\.reform\.hmcts\.net|dm-store-prod\.service\.core-compute-prod\.internal(?::\d+)?)
+        CCD_DOCUMENT_URL_PATTERN: ^https?://(?:api-gateway\.preprod\.dm\.reform\.hmcts\.net|(dm-store|em-hrs-api)-prod\.service\.core-compute-prod\.internal(?::\d+)?)\/(documents\/[A-Za-z0-9-]+(?:\/binary)?)|(hearing-recordings\/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\/segments\/[0-9])
```

Comment:
pls raise a separate PR for production. We will merge that only when the service is ready.